notebooks/Tutorial_Top-K_Multidimensional_Motif_and_Matches_Discovery.ipynb

```diff
@@ -0,0 +1,242 @@
+{
```

Comment:
<p>I don't think that it is wise to have a function with no inputs as this can cause unintended behavior. Instead, you should always pass in the latest <code>motif_distances</code>, <code>motif_distances</code> , <code>df</code>, etc.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/557/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Ok, thank you, I'll change it!

Reply:
Unfortunately this doesn't change anything and I still get different motifs. I have to take a closer look at this the next days. I don't like Jupyter notebooks since I am often debugging my way through all the code which is inconvenient with them in my opinion. So I resort to copy-pasting it to Pycharm and back as needed 😆